src/App.tsx

```diff
@@ -15,48 +17,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPABET,
+  LENGTH,
+}
+
+export function getReorderedGoods(
```

Comment:
it is a bad practice to accept a lot of arguments in the function, it makes your code more complicated and less possible to reuse later